github/resource_github_branch_protection_test.go

```diff
@@ -56,8 +54,6 @@ func TestAccGithubBranchProtection_basic(t *testing.T) {
 					testAccCheckGithubBranchProtectionNoPullRequestReviewsExist(&protection),
 					resource.TestCheckResourceAttr(rn, "repository", repoName),
 					resource.TestCheckResourceAttr(rn, "branch", "master"),
-					resource.TestCheckResourceAttr(rn, "require_signed_commits", "false"),
-					resource.TestCheckResourceAttr(rn, "required_status_checks.0.strict", "false"),
```

Comment:
might of accidentally removed this line for `required_status_checks`?

Reply:
Whoops, nice catch. Added it again, thanks!